lib/services/presets.go

```diff
@@ -218,6 +218,9 @@ func NewPresetEditorRole() types.Role {
 					types.NewRule(types.KindSigstorePolicy, RW()),
 					types.NewRule(types.KindWorkloadIdentityX509IssuerOverride, RW()),
 					types.NewRule(types.KindWorkloadIdentityX509IssuerOverrideCSR, RW()),
+					types.NewRule(types.KindSummarizationInferenceModel, RW()),
+					types.NewRule(types.KindSummarizationInferenceSecret, RW()),
+					types.NewRule(types.KindSummarizationInferencePolicy, RW()),
```

Comment:
Are these the actual yaml configuration resources a user is supposed to be able to create/edit? I don't see their protos in this PR. And also if so, why do we have 3 new resources, can't we have a single "summarization inference config" or something to simplify?